### PR TITLE
edge case handling for separations

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**
no ETA or TED for Separations

### CODE REVIEW

File: talentmap_api/fsbid/services/agenda.py [hunk 1]
```diff
@@ -883,8 +884,8 @@ def convert_agenda_item_leg_query(query, leg={}):
         "ailtodcode": pydash.get(leg, "tod", ""),
         "ailtodmonthsnum": tod_months if is_other_tod else None, # only custom/other TOD should pass back months and other_text
         "ailtodothertext": tod_long_desc if is_other_tod else None, # only custom/other TOD should pass back months and other_text
-        "ailetadate": eta[:eta.rfind(".000Z")],
-        "ailetdtedsepdate": ted[:ted.rfind(".000Z")],
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/agenda.py:887]: rfind returns -1 if not found

File: talentmap_api/fsbid/services/agenda.py [hunk 2]
```diff
@@ -172,7 +172,7 @@ def modify_agenda(query={}, jwt_token=None, host=None):
                         (asg_seq_num != existing_asg_seq_num) or
                         (asg_revision_num != existing_asg_revision_num)
                     ):
-                        agenda_item = edit_agenda_item(query, jwt_token)
+                        edit_agenda_item(query, jwt_token)
```
Comment (scott062) [inline talentmap_api/fsbid/services/agenda.py:175]: nice catch

File: talentmap_api/fsbid/services/agenda.py (hunk 1 above)
Comment (scott062) [reply talentmap_api/fsbid/services/agenda.py:887]: How dare you - greta gif

**Review by scott062 — APPROVED** [review]

File: talentmap_api/fsbid/services/agenda.py (hunk 1 above)
Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/agenda.py:887]: wow wow

File: talentmap_api/fsbid/services/agenda.py (hunk 2 above)
Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/agenda.py:175]: it was actually just confusing me 😂 so just making it easier for myself next time

**Pull request merged**